spock-core/src/main/java/org/spockframework/compiler/model/BlockParseInfo.java

```diff
@@ -16,7 +16,7 @@
 
 package org.spockframework.compiler.model;
 
-import java.util.EnumSet;
+import java.util.*;
```

Comment:
Why this change for just one import?

Reply:
It's not just one import `java.util.Locale` was added.